src/CoreText/CTLine.cs

```diff
@@ -233,7 +233,7 @@ public nfloat GetOffsetForStringIndex (nint charIndex)
 		}
 
 		public delegate void CaretEdgeEnumerator (double offset, nint charIndex, bool leadingEdge, ref bool stop);
-		unsafe delegate void CaretEdgeEnumeratorProxy (IntPtr block, double offset, nint charIndex, bool leadingEdge, ref bool stop);
+		unsafe delegate void CaretEdgeEnumeratorProxy (IntPtr block, double offset, nint charIndex, [MarshalAs (UnmanagedType.I1)] bool leadingEdge, [MarshalAs (UnmanagedType.I1)] ref bool stop);
```

Comment:
I find it interesting that this is only needed on the delegate, and not the function getting called (but it obviously works, since the tests passed).